common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -553,11 +562,6 @@
                         self.loaded = true;
                     },
                     error: function (jqXHR, textStatus, errorThrown) {
-                        console.log('[Video info]: ERROR while fetching captions.');
```

Comment:
These logs seem like they might be useful.  Are you sure we should be removing them?